components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_switch.xml

```diff
@@ -15,7 +15,7 @@
         style="@style/Mozac.Browser.Menu.Item.ImageText.Icon"
         android:background="@android:color/transparent"
         android:clickable="false"
-        tools:src="@android:drawable/screen_background_dark" />
+        app:srcCompat="@android:drawable/screen_background_dark" />
```

Comment:
Let's not change the `tools:src` attributes. This is using the `tools` namespace since we only want to display those items in the layout preview of Android Studio and not actually assign them.